instrumentation/apache-dubbo-2.7/library-autoconfigure/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/TracingFilter.java

```diff
@@ -63,7 +63,7 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) {
       throw e;
     }
     if (isSynchronous) {
-      instrumenter.end(context, request, result, null);
+      instrumenter.end(context, request, result, result.getException());
```

Comment:
should async case also use something like `throwable != null ? throwable : result.getException()`?

Reply:
this change is only for server side. the async case is for the client in dubbo. I check the code in brave too, it only use 'result.getException()' in sync case too.